packages/x-data-grid-pro/src/hooks/utils/useGridAriaAttributes.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { useGridAriaAttributes as useGridAriaAttributesCommunity } from '@mui/x-data-grid/internals';
+import { useGridRootProps } from './useGridRootProps';
+
+export const useGridAriaAttributes = (): React.HTMLAttributes<HTMLElement> => {
+  const ariaAttributesCommunity = useGridAriaAttributesCommunity();
+  const rootProps = useGridRootProps();
+
+  const ariaAttributesPro = rootProps.treeData ? { role: 'treegrid' } : {};
```

Comment:
Should this also depend on `ariaV8` flag?
If yes, maybe add a section like [this](https://deploy-preview-13623--material-ui-x.netlify.app/x/react-data-grid/row-grouping/#accessibility-changes-in-v8) in tree data documentation [page](https://deploy-preview-13623--material-ui-x.netlify.app/x/react-data-grid/tree-data/) too?

Reply:
Pro attribute was already there, but it was added via the community gird hook
https://github.com/mui/mui-x/blob/51ec133c8d19d621d660f9c0864495d99dabea32/packages/x-data-grid/src/hooks/utils/useGridAriaAttributes.tsx#L21-L23